vaadin-ordered-layout-flow-parent/vaadin-ordered-layout-flow/src/main/java/com/vaadin/flow/component/orderedlayout/FlexComponent.java

```diff
@@ -289,6 +289,59 @@ default public double getFlexGrow(HasElement elementContainer) {
         }
     }
 
+    /**
+     * Sets the flex shrink property of the components inside the layout. The
+     * flex shrink property specifies how the item will shrink relative to the
+     * rest of the components inside the same layout.
+     *
+     * Negative values are not allowed.
+     *
+     * The default value is 1.
+     *
+     * @param flexShrink
+     *            how much the component will shrink relative to the rest of the
+     *            components
+     * @param elementContainers
```

Comment:
Would it make sense to rename this parameter? Reading this I was confused why I have to pass the Flex container as parameter.

It could be `components`, or `children`. Even though it's technically not accurate, it seems a lot more helphul.

Reply:
Looks like the confusing naming originates from https://github.com/vaadin/vaadin-ordered-layout-flow/pull/46 which addressed https://github.com/vaadin/flow/issues/3511

In order to keep the focus of this PR in relocating the method, I'll create a separate PR with the [parameter renaming changes](https://github.com/vaadin/flow-components/commit/4fec2bcc44335a53fa90692791f1932a12dcc99e) separately.

Reply:
Sure, thanks 👍